site/about/glossary/_developer-tools.qmd

```diff
@@ -0,0 +1,33 @@
+decorator, Python decorator
+: A [design pattern in Python](https://www.datacamp.com/tutorial/decorators-python) that allows a user to add new functionality to an existing object without modifying its structure.
+
+  - Decorators are a simpler way for users to run their
+own code as a ValidMind test.
+
+{{< include key_concepts/_inputs.qmd >}}
+
+{{< include key_concepts/_outputs.qmd >}}
+
+{{< include key_concepts/_parameters.qmd >}}
+
+pip
+: A package manager for Python, used to install and manage software packages written in the Python programming language. 
+
+  - ValidMind uses the `pip` command to install the Python client library that is part of the {{< var vm_framework >}} so that model developers can make use of its features.
+
+JupyterHub
+: A multi-user server provides a platform for users to interactively work with data science and scientific computing tools in a collaborative environment. 
+
+  - ValidMind uses JupyterHub to share live code, how-to instructions, and visualizations via notebooks as part of our getting started experience for new users.
+
+Jupyter notebook
+: Allows users to create and share documents containing live code, data visualizations, and narrative text. Supports various programming languages, most notably Python, and is widely used for data analysis, machine learning, scientific research, and educational purposes. 
+
+  - ValidMind uses notebooks to share sample code and how-to instructions with users that you can adapt to your own use case.
+
+{{< include key_concepts/_metrics.qmd >}}
+
+GitHub
+: A cloud-based platform that provides hosting for software development and version control using Git. [GitHub](https://github.com/) offers collaboration tools such as bug tracking, feature requests, task management, and continuous integration pipelines. 
+
+  - ValidMind uses GitHub to share [open-source software](https://github.com/validmind/developer-framework/) with you.
```

Comment:
```suggestion
  - ValidMind uses GitHub to share [open-source software](https://github.com/validmind/) with you.
```

We have an organization-level readme we can point people to and we already have more than one open-source repo.